src/applications/disability-benefits/686c-674/config/chapters/add-a-child/child-place-of-birth/childStatusDescription.jsx

```diff
@@ -11,40 +11,34 @@ export const childStatusDescription = (
       or if you don’t live in the U.S. or a territory of the U.S., we require
       additional evidence to establish a dependent.
     </p>
-    <AdditionalInfo triggerText="What are some examples of additional evidence?">
-      <p>
-        To add a stepchild, you’ll need to provide a copy of their birth
-        certificate, showing the names of both parents if:
+    <AdditionalInfo triggerText="Additional evidence needed to add children">
+      <p className="vads-u-margin-bottom--4">
+        If you <strong>don’t live in the U.S.</strong> or a territory of the
+        U.S., you’ll need to provide a copy of each child’s birth certificate
       </p>
-      <ul>
-        <li>
-          Your stepchild is not the biological child of your spouse (adopted,
-          for example) or
-        </li>
-        <li>You don’t live in the U.S. or a territory of the U.S.</li>
-      </ul>
-      <p>
-        A claim for additional benefits for an <strong>adopted child</strong>{' '}
-        must include a copy of one of the following:
+      <p className="vads-u-margin-bottom--4">
+        To add a <strong>stepchild,</strong> you’ll need to provide a copy of
```

Comment:
@cohnjesse - do you mind confirming with @sporeboy and @jason-gcio about this language change? I know we talked recently (within the last two weeks I think) about not needing a birth certificate if a stepchild is the biological child of a spouse.